Visibility on jobs for May updates (#619)
akeneo/pim-helpcenter: content/updates/2020-05/3-visibility-on-jobs.md

# More visibility on PIM jobs
::: meta-data type="Improvement" features="Productivity, Process tracker" available="mid-May" link-to-doc="../articles/monitor-jobs.html#how-to-monitor"
:::

> To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. It will enable you to see all jobs of any status under the Process Tracker. You will now be able to check if you have any ongoing running jobs and which users launched them . :wink:

Reviewer: ```suggestion
To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. It will enable you to see all jobs of any status under the Process Tracker. You will now be able to see the ongoing or previous PIM jobs and if needed which users launched them. :wink:
```

# More visibility on all the jobs
::: meta-data type="Improvement" features="Productivity, Process tracker" available="mid-May" link-to-doc="../articles/monitor-jobs.html#how-to-monitor"
:::

> Say welcome to a brand new permission called `View all jobs in the Process Tracker`! It will allow you to see all jobs in the Process Tracker, even the ones that you didn't launch. Really neat if you are an admin and need to have a quick overview of what's running in the PIM. :wink:

Reviewer 2: ```suggestion
Say welcome to a brand new permission called `View all jobs in the Process Tracker`! It will allow you to see all jobs in the Process Tracker, even the ones that you didn't launch. Really neat if you are an admin and need to have a quick overview of what's going on in the PIM. :wink:
```

> Say welcome to a brand new permission called `View all jobs in the Process Tracker`! It will allow you to see all jobs in the Process Tracker, even the ones that you didn't launch. Really neat if you are an admin and need to have a quick overview of what's running in the PIM. :wink:

Reviewer: ```suggestion
To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. It will enable you to see all jobs of any status under the Process Tracker. You will now be able to check if you have any ongoing running jobs and which users launched them . :wink:
```

> Say welcome to a brand new permission called `View all jobs in the Process Tracker`! It will allow you to see all jobs in the Process Tracker, even the ones that you didn't launch. Really neat if you are an admin and need to have a quick overview of what's going on in the PIM. :wink:

Reviewer: To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. It will enable you to see all jobs of any status under the Process Tracker. You will now be able to check if you have any ongoing running jobs and which users launched them . :wink:

> The previous `View Process Tracker` permission still exists and can now be combined with this new permission.

Reviewer: Maybe we could remove this part..? Isn't it too detailed?

Author: We can. 
```suggestion
```
